contracts/CO30_Matias_FlightOfIcarus.Altis/mission.sqm

```diff
@@ -0,0 +1,10256 @@
+					};
+					id=695;
+					type="O_R_Soldier_GL_F";
+				};
+				class Item5
+				{
+					dataType="Object";
+					class PositionInfo
+					{
+						position[]={4808.2178,252.31499,21489.516};
+						angles[]={0,4.3675642,0};
+					};
+					side="East";
+					flags=5;
+					class Attributes
+					{
+						skill=0.44999999;
+						rank="CORPORAL";
+						stance="Up";
+					};
+					id=696;
+					type="O_R_Soldier_AR_F";
+				};
+				class Item6
+				{
+					dataType="Object";
+					class PositionInfo
+					{
+						position[]={4818.3662,244.3342,21461.275};
+						angles[]={0,4.3675642,0};
+					};
+					side="East";
+					flags=5;
+					class Attributes
+					{
+						skill=0.40000001;
+						stance="Up";
+					};
+					id=697;
+					type="O_R_Soldier_LAT_F";
+				};
+				class Item7
+				{
+					dataType="Object";
+					class PositionInfo
+					{
+						position[]={4811.2344,254.97685,21495.902};
+						angles[]={0,4.3675642,0};
+					};
+					side="East";
+					flags=5;
+					class Attributes
+					{
+						skill=0.40000001;
+						stance="Up";
+					};
+					id=698;
+					type="O_R_soldier_M_F";
+				};
+				class Item8
+				{
+					dataType="Waypoint";
+					loiterAltitude=-3.4028235e+038;
+					position[]={4793.5591,246.3232,21471.725};
+					type="Hold";
+					class Effects
+					{
+					};
+					showWP="NEVER";
+					id=699;
+					atlOffset=3.0517578e-005;
+				};
+				class Item9
+				{
+					dataType="Waypoint";
+					loiterAltitude=-3.4028235e+038;
+					position[]={4733.8911,253.98399,21486.936};
+					class Effects
+					{
+					};
+					showWP="NEVER";
+					id=700;
+					type="Move";
+					atlOffset=0.0001373291;
+				};
+				class Item10
+				{
+					dataType="Waypoint";
+					loiterAltitude=-3.4028235e+038;
+					position[]={4596.2915,298.71634,21427.52};
+					class Effects
+					{
+					};
+					showWP="NEVER";
+					id=701;
+					type="Move";
+					atlOffset=-9.1552734e-005;
+				};
+				class Item11
+				{
+					dataType="Waypoint";
+					loiterAltitude=-3.4028235e+038;
+					position[]={4577.3232,300.33075,21427.533};
+					type="lambs_danger_Hunt";
+					script="\z\lambs\addons\wp\scripts\fnc_wpHunt.sqf";
+					class Effects
+					{
+					};
+					showWP="NEVER";
+					id=702;
+					atlOffset=-6.1035156e-005;
+				};
+			};
+			class Attributes
+			{
+			};
+			id=690;
+			atlOffset=3.0517578e-005;
+		};
+		class Item56
+		{
+			dataType="Object";
+			class PositionInfo
+			{
+				position[]={4557.5889,304.83472,21493.221};
+				angles[]={0,5.4524612,0};
+			};
+			side="Empty";
+			class Attributes
+			{
+				name="church_radio";
+			};
+			id=711;
+			type="Land_PortableLongRangeRadio_F";
+			atlOffset=0.19995117;
+		};
+		class Item57
+		{
+			dataType="Trigger";
+			position[]={4557.4761,304.81927,21492.238};
+			class Attributes
+			{
+				sizeA=0.688187;
+				sizeB=1.6184556;
+				sizeC=5;
+				activationBy="ANYPLAYER";
+				isRectangle=1;
+				isServerOnly=1;
+				effectSoundTrigger="Zeus_Sound_Rus_Radio";
```

Comment:
Is this trigger supposed to have a sound effect? No one will be able to hear it since the trigger is server only.

Reply:
Didn't know that worked liked that

Reply:
Remember that when testing in the editor, you are the server and the client.